### PR TITLE
Ability to manually loading withdraw with external txid

### LINKED ISSUE #1864: Ability to manually loading withdrawal with external txid from admin panel

### PR DESCRIPTION
**Pull request by mnaichuk**
closes #1864

### CODE REVIEW

File: app/controllers/admin/withdraws/coins_controller.rb [hunk 1]
```diff
@@ -9,30 +9,69 @@ class CoinsController < BaseController
...
+        end
+      end
+
+      def show
+
+      end
+
+      def update
+        case params.fetch(:event)
+        when 'process'
```
Comment (ysv) [inline app/controllers/admin/withdraws/coins_controller.rb:28]:
```suggestion
        when 'process'
        process!
        when 'load'
        load!
```

File: docs/api/member_api_v2.md [hunk 2]
```diff
@@ -2,7 +2,7 @@ Member API v2
 =============
 Member API is API which can be used by client application like SPA.
 
-**Version:** 2.0.0-alpha
+**Version:** 1.9.1-rc.27
```
Comment (ysv) [inline docs/api/member_api_v2.md]: rebase please

File: app/controllers/admin/withdraws/coins_controller.rb [hunk 3]
```diff
@@ -9,30 +9,69 @@ class CoinsController < BaseController
       before_action :find_withdraw, only: [:show, :update, :destroy]
 
       def index
-        @latest_withdraws  = ::Withdraws::Coin.where(currency: currency)
-                                              .where('created_at <= ?', 1.day.ago)
+        case params.fetch(:state, 'all')
```
Comment (ysv) [inline app/controllers/admin/withdraws/coins_controller.rb]: I think we need specs for this controller

**Review by ysv — CHANGES_REQUESTED** [review]

**Comment by pkucherenk0** [comment]
Add ability to reject already confirming withdraw in case of input tx id error.
Currently the amount will be blocked forever.

**Pull request merged**